Src/Model.cpp

```diff
@@ -208,6 +229,135 @@ std::unique_ptr<EffectTextureFactory> Model::LoadTextures(
 }
 
 
+// Load VB/IB resources for static geometry
+_Use_decl_annotations_
+void Model::LoadBuffers(
```

Comment:
This is the algorithm which converts a Model using dynamic VB/IB to static VB/IB. If you pass keepMemory = true (it defaults to false) I will keep the original ``vertexBuffer`` and ``indexBuffer`` members set as well as setting the ``staticVertexBuffer`` and ``staticIndexBuffer`` values. The rendering will only use the static versions if both are set.